script.js

```diff
@@ -1,12 +1,64 @@
+/* Homepage Functions */
+function on() {
+  document.getElementById("overlay").style.display = "block";
+}
 
+function off() {
+  document.getElementById("overlay").style.display = "none";
+}
 
-function on() {
-    document.getElementById("overlay").style.display = "block";
-  }
-  
-  function off() {
-    document.getElementById("overlay").style.display = "none";
-  }
+/* Dynamic Project Functions */
+function projectOn() {
+  document.getElementById("output").style.display = "block";
+}
+
+function projectOff() {
+  document.getElementById("output").style.display = "none";
+}
 
+
+
+function getProject(a) {
+  projectOn();
+fetch("projects.json")
+.then((res) => res.json())
+.then((data) => {
+ output = `
+ <div class="project_overlay">
+  <div class="project_structure_overlay">
+      <div> 
+          <button onclick="projectOff()" class="project_overlay_btn_close"> X </button>
+      </div>
+      <div class="project_heading_overlay">
+              <h2 class="title_overlay"> ${data[a].name} </h2>
+              <ul class="project_body_title_list">
+                  <li class="title_list_main_overlay">${data[a].projectClient}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[0]}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[1]}</li>
+              </ul>
+      </div>
+      <div> <img class="image_ovlerlay" src=${data[a].image} alt="snapshoot"> </div>
+      <div class="project_body_overlay">
+          <div class="text_group_overlay">
+              <p>${data[a].mainText}</p>
+          </div>
+          <div class="btn_group_overlay">
+          <ul class="project_tech_overlay">
+                  <li>${data[a].technologies[0]}</li>
+                  <li>${data[a].technologies[1]}</li>
+                  <li>${data[a].technologies[2]}</li>
+              </ul>
+              <hr class="hr_style">
+              <div class="btn_flex_overlay">
+                <button class="overlay_btn"> <a href="${data[a].liveVersionLink}"> See Live <img src="./Assets/Icon_Export.png" > </a> </button>
+                <button class="overlay_btn"> <a href="${data[a].liveVersionLink}"> See Source <img src="./Assets/github_in_button.png" > </a> </button>
```

Comment:
- [ ] For the see live button and see source button I think it would be great if you increase the width for the icon beside the text to let the user be able to see it that is very important for user expereince 🚀♻👍🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/202315071-e6ff91b8-2693-4479-9dad-3ba11977b72e.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/202315188-601cdf85-af85-4cb0-87c7-fb3ef6ec5cca.png)